src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -30,7 +30,7 @@ public static class ActivityExtensions
         /// <summary>
         /// Sets the status of activity execution.
         /// Activity class in .NET does not support 'Status'.
-        /// This extension provides a workaround to store Status as special tags with key name of ot.status_code and ot.status_description.
+        /// This extension provides a workaround to store Status as special tags with key name of otel.status_code and otel.status_description.
```

Comment:
can you update changelog for this change?

Reply:
just updated the api/changelog.